package.json

```diff
@@ -15,7 +15,9 @@
     "dist"
   ],
   "dependencies": {
+    "@types/lodash": "^4.14.119",
```

Comment:
I'd like to use the more specific `@types/lodash.throttle` package to prevent version conflicts